securedrop/manage.py

```diff
@@ -210,6 +248,51 @@ def listdir_fullpath(d):
     return 0
 
 
+def translate(args):
+    messages_file = os.path.join(args.translations_dir, 'messages.pot')
+
+    if args.extract_update:
+        sh("""
+        set -xe
+
+        mkdir -p {translations_dir}
+
+        pybabel extract \
+        --charset=utf-8 \
+        --mapping={mapping} \
+        --output={messages_file} \
+        --project=SecureDrop \
+        --version={version} \
+        --msgid-bugs-address='securedrop@freedom.press' \
+        --copyright-holder='Freedom of the Press Foundation' \
+        {sources}
+
+        # we do not handle fuzzy translations yet
+        sed -i '/^#, fuzzy$/d' {messages_file}
+
+        # stop here if there are no translations (for testing)
+        test $(ls {translations_dir} | wc -l) = 1 && exit 0
```

Comment:
For these test lines can we either: do this in Python e.g. with `len(os.listdir(translations_dir)` and print a message if we exit prematurely on this line i.e. "Exited - No translations found!" or similar to do the same in bash, e.g. add a `|| print "Exited - No translations found!"`.